recipes/rsconnect-python/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "rsconnect-python" %}
+{% set version = "1.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rstudio/rsconnect-python/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 709d584ca256fb13ae6f53f9b2b5aa96e71f151dd94a83249b7f82d671ee37d5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
```

Comment:
Upstream `install_requires` says `click >=7.0.0` and `six >=1.14.0`. Any reason not to match?

Reply:
Is the correct way to mirror over the requirements.txt over to host? and also to run if we expect the user as host?

Reply:
added with https://github.com/conda-forge/staged-recipes/pull/14506/commits/613a5adbc78caeeb112827af85c80150dcf92815